jax/_src/numpy/reductions.py

```diff
@@ -785,10 +785,81 @@ def _average(a: ArrayLike, axis: Axis = None, weights: ArrayLike | None = None,
   return avg
 
 
-@implements(np.var, skip_params=['out'])
 def var(a: ArrayLike, axis: Axis = None, dtype: DTypeLike | None = None,
         out: None = None, ddof: int = 0, keepdims: bool = False, *,
         where: ArrayLike | None = None, correction: int | float | None = None) -> Array:
+  r"""Compute the variance along a given axis.
+
+  JAX implementation of :func:`numpy.var`.
+
+  Args:
+    a: Input array.
+    axis: int or array, default=None. Axis along which the variance to be computed.
+      If None, variance is computed along all the axes.
+    dtype: The type of the output array. Default=None.
+    ddof: int, default=0. Degrees of freedom. The divisor in variance computation is
+      ``N-ddof``, ``N`` is number of elements along given axis.
+    keepdims: bool, default=False. If true, reduced axes are left in the result
+      with size 1.
+    where: int or array, default=None. The elements to be used in the variance.
```

Comment:
`where` can't be an int
```suggestion
    where: optional boolean array, default=None. The elements to be used in the variance.
```

Reply:
Done